tf/persistent/README.md

```diff
@@ -1,7 +1,118 @@
-# Persistent testing resources
+## Persistent testing resources
 
-This module is for creating resources that should be persistent across tests.
-Right now it manages:
+This module is for creating resources that should be persistent across tests. There are no automated tests for the terraform files in this directory. 
 
-- The GKE cluster for running test pods
-- The Cloud Build CI Triggers for each repo
+Most of the resources used by these tests are set up and torn down for each test run. There are however some persistent resources which are managed by terraform (under [tf/persistent](../persistent/)) in the default terraform workspace:
+
+- The GKE cluster ([e2etest-default](https://pantheon.corp.google.com/kubernetes/clusters/details/us-central1/e2etest-default/details?project=opentelemetry-ops-e2e)) for running test pods.
+- The Cloud Build CI Triggers for each repo ([pantheon link](https://pantheon.corp.google.com/cloud-build/triggers?project=opentelemetry-ops-e2e)).
+- The default service for the created GAE application ([pantheon link](https://pantheon.corp.google.com/appengine/services?serviceId=default&versionId=v1&project=opentelemetry-ops-e2e)).
```

Comment:
I don't think we want to link to our actual project in these docs, or if we do it shouldn't be a `*.corp.google.com` link